be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -117,12 +119,12 @@ Status VOlapTableSink::send(RuntimeState* state, vectorized::Block* input_block)
         _partition_to_tablet_map.clear();
     }
     
-    //if pending bytes is more than 500M, wait
+    //if pending bytes is more than 500M, abort task
     constexpr size_t MAX_PENDING_BYTES = 500 * 1024 * 1024;
-    if ( get_pending_bytes() > MAX_PENDING_BYTES){
-        while(get_pending_bytes() < MAX_PENDING_BYTES){
-            std::this_thread::sleep_for(std::chrono::microseconds(500));
-        }
+    if (get_pending_bytes() > MAX_PENDING_BYTES) {
```

Comment:
Not a good fix.
If the data size in a single batch is larger than 500MB, user can not load data anymore.